dtests/src/test/java/io/snappydata/hydra/longRun/longRunTestEmbededMode.conf

```diff
@@ -0,0 +1,134 @@
+hydra.Prms-testRequirement = "Test to verify functionality in embeded mode while running the cluster on long duration with different schemas";
+hydra.Prms-testDescription = "This test starts the snappy cluster, initializes snappyContext, create tables using snappy job. 
```

Comment:
You are using sql script for creating tables. Modify the description accordingly.